src/app/projects/components/data-use-restriction-group/data-use-restriction-group.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, OnInit } from '@angular/core';
+import {
+  AbstractControl,
+  FormBuilder,
+  FormGroup,
+  ValidatorFn,
+  Validators
+} from '@angular/forms';
+import { MetadataForm } from '@metadata-schema-form/models/metadata-form';
+import { MetadataFormHelper } from '@metadata-schema-form/models/metadata-form-helper'
+
+@Component({
+  selector: 'app-data-use-restriction-group',
+  templateUrl: './data-use-restriction-group.component.html',
+  styleUrls: ['./data-use-restriction-group.component.css']
+})
+
+export class DataUseRestrictionGroupComponent implements OnInit {
+  metadataForm: MetadataForm;
+  form: FormGroup;
+  formHelper: MetadataFormHelper;
+
+  constructor(private fb: FormBuilder) {}
+
+  ngOnInit(): void {
+    this.formHelper = new MetadataFormHelper();
+    this.initializeForms();
+    this.subscribeToDataUseRestrictionChanges();
+  }
+
+  initializeForms(): void {
+    const dataUseRestrictionMetadata = this.metadataForm.get('project.content.data_use_restriction');
+
+    // Ignore example values for DUO codes
+    if (dataUseRestrictionMetadata) {
+      dataUseRestrictionMetadata.ignoreExample = true;
+    }
```

Comment:
One clean code approach that improves readability and comment rot would be to create functions for the logical steps in the function which are described in the comments.

In other words, you could create a function called `ignoreExampleValues`.
This function is rather short, so it would not be mandatory. It is a good practice for longer functions. Apart from the aforementioned advantages, it also allows testing individual steps in a complex logic.